index.html

```diff
@@ -32,15 +31,195 @@
+                    <div class="connect">
+                        <h5>LET'S CONNECT</h5>
+                    </div>
+                </div>
+                <ul class="social">
+                    <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+                </ul>
+                <div class="btn_see_resume_contain" >
+                    <a href="#" class="btn-see-resume">Get my resume</a>
                 </div>
+                <ul class="main-skills">
+                    <li>
+                        <ul class="skills">
+                        <li class="header-skill">
+                            Languages
+                            <i class="fa fa-chevron-down push"></i>
+                        </li>
+                        <li>
+                            <div class="dflex">
+                            <div class="dflex-item"><img src="assets/images/js.png" alt="javascript" /></div>
+                            <div class="dflex-item"> Javascript</div>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="dflex">
+                            <div class="dflex-item"><img src="assets/images/html.png" alt="javascript" /></div>
+                            <div class="dflex-item"> HTML</div>
+                            </div>
+                        </li>
+                        <li class="dflex">
+                            <div class="dflex-item"><img src="assets/images/css.png" alt="javascript" /></div>
+                            <div class="dflex-item"> CSS</div>
+                        </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <ul class="more_skills">
+                        <li class="lined_skill">
+                            <span>Frameworks</span>
+                            <span><i class="fa fa-chevron-right push"></i></span>
+                        </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <ul class="more_skills">
+                        <li class="lined_skill">
+                            Skills
+                            <i class="fa fa-chevron-right push"></i>
+                        </li>
+                        </ul>
+                    </li>
+                    </ul>
```

Comment:
- [ ] By the look of things, your design is having some list style that is not present in the original design. Try to set the list style to none in your CSS file. More also, your font sizing and font family needs to match with the given design.

Your Design                                                        |  Figma Design                                |
:-------------------------------------------------------------:|:-----------------------------------------------:
![Your Design](https://user-images.githubusercontent.com/39523312/143503630-c45d7c7a-c2eb-485c-9da6-e0323aec7a2e.png)   |  ![Figma Design](https://user-images.githubusercontent.com/39523312/143503633-9039c52a-dc23-42f9-9507-0211ae8dd2c8.png)|